src/main/resources/defaults/displayViews/documents/overview/nameOfConference.html

```diff
@@ -0,0 +1 @@
+<p>{{{nameOfConference}}}</p>
```

Comment:
Are we following the standard practice of newlines at the end of a file?
If so, can you add a newline to this and the other html file?